x/incentives/types/msgs.go

```diff
@@ -55,8 +58,21 @@ func (m MsgCreateGauge) ValidateBasic() error {
 		return errors.New("distribution period should be 1 epoch for perpetual gauge")
 	}
 
-	if lockuptypes.LockQueryType_name[int32(m.DistributeTo.LockQueryType)] != "ByDuration" {
-		return errors.New("only duration query condition is allowed. Start time distr conditions is an obsolete codepath slated for deletion")
+	if lockType == lockuptypes.ByTime {
```

Comment:
What do you think of completely getting rid of `lockuptypes.ByTime`. Definitely out of scope for this PR but we can create and issue and track it

Reply:
I don't think we can remove it for backward compatibility concerns. We might have to do a migration to remove it. If there were a trivial way to do it, I would be fully supportive